framework/src/st_things/things_stack/things_stack.c

```diff
@@ -62,7 +62,7 @@
 #endif
 
 #define TAG "[things_stack]"
-
+#define scan_ap_interval 60
```

Comment:
All MACROs should be in UPPER_CASE. Change it.

Reply:
fixed.